doc/user_guide/electron_holography.rst

```diff
@@ -0,0 +1,42 @@
+Electron Holography
+*******************
+
+HyperSpy provides the user with a class which can be used to process electron holography data and
+electron wave data in general:
+
+* :py:class:`~._signals.electron_wave_image.ElectronWaveImage`
+
+The usage of the class is explained in the following sections.
+
+
+
+The ElectronWaveImage class
+===========================
+
+The :py:class:`~._signals.electron_wave_image.ElectronWaveImage` class can hold information about
+the complex electron wave. It inherits from :py:class:`~._signals.complex_signal.ComplexSignal`
+and as such relevant properties like the `amplitude`, `phase` and the `real` and `imag` part of
+the complex data can be directly accessed and return appropriate
+:py:class:`~._signals.signal2d.Signal2D` signals.
+
+To transform a complex signal into a :py:class:`~._signals.electron_wave_image.ElectronWaveImage`
+use:
+
+.. code-block:: python
+
+    >>> s.metadata.Signal.record_by = 'image'
+    >>> s.set_signal_type('electron wave')
+
+Note that a :py:class:`~._signals.electron_wave_image.ElectronWaveImage` does not inherit from
+:py:class:`~._signals.signal2d.Signal2D`, because most functionality can not operate on complex
+data.
+
+
+Add a linear phase ramp
+-----------------------
+
+A linear phase ramp can be added to the signal via the :py:func:`~._signals.signal2d.Signal2D.add_phase_ramp`

```

Comment:
I know that there is no `ComplexSignal2D` yet, but, maybe `add_phase_ramp` could be moved to a new `ComplexSignal2D`?